uwsdora.module

```diff
@@ -106,3 +106,54 @@ function uwsdora_views_api() {
     'path' => drupal_get_path('module', 'uwsdora') . '/views',
   );
 }
+
+/**
+ * Implements hook_convert_mods_to_citeproc_jsons_alter().
+ *
+ * @see citeproc/includes/converter.inc::convert_mods_to_citeproc_json_page()
```

Comment:
I'd actually move this down to the anonymous function (either inside or just before it), since that's where this in particular is relevant.